components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -574,6 +639,13 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
         claimsMap.putAll(getSubjectAttributes(oAuthResponse, authenticatorProperties));
         authenticatedUser.setUserAttributes(claimsMap);
         context.setSubject(authenticatedUser);
+        if (LoggerUtils.isDiagnosticLogsEnabled() && diagnosticLogBuilder != null) {
+            diagnosticLogBuilder.resultMessage("Successfully completed the outbound OIDC authentication response.")
```

Comment:
Let's reword the message. 
eg: Outbound authentication OIDC response processed successfully

Reply:
Fixed with 4d93f2eea213591f5dbd1864f05968e280897363